django/contrib/admin/sites.py

```diff
@@ -63,6 +64,21 @@ def __init__(self, name='admin'):
         self.name = name
         self._actions = {'delete_selected': actions.delete_selected}
         self._global_actions = self._actions.copy()
+        all_sites.add(self)
+
+    def check(self):
+        """
+        Run the system checks of the underlying admin objects.
+        """
+        if not settings.DEBUG:
+            return []
+
+        errors = []
+        admin_objects_to_check = [o for o in self._registry.values() if o.__class__ is not ModelAdmin]
```

Comment:
@ellmetha some checks look at the suitability of the model class for the admin, rather than just attributes on `ModelAdmin`, it's rather presumptive to skip them...